# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux/hotfix-ia64-pagesize.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-6.11/arch/ia64/Kconfig.vanilla	2024-10-05 15:41:25.893014099 +0200
+++ linux-6.11/arch/ia64/Kconfig	2024-10-05 15:44:24.716343525 +0200
@@ -143,6 +143,8 @@
 
 config IA64_PAGE_SIZE_4KB
 	bool "4KB"
+	select HAVE_PAGE_SIZE_4KB
+	select PAGE_SIZE_4KB
 	help
 	  This lets you select the page size of the kernel.  For best IA-64
 	  performance, a page size of 8KB or 16KB is recommended.  For best
@@ -159,13 +161,18 @@
 	  If you don't know what to do, choose 16KB.
 
 config IA64_PAGE_SIZE_8KB
+	select HAVE_PAGE_SIZE_8KB
+	select PAGE_SIZE_8KB
 	bool "8KB"
 
 config IA64_PAGE_SIZE_16KB
+	select HAVE_PAGE_SIZE_16KB
+	select PAGE_SIZE_16KB
 	bool "16KB"
 
 config IA64_PAGE_SIZE_64KB
 	depends on !ITANIUM
+	select HAVE_PAGE_SIZE_64KB
 	select PAGE_SIZE_64KB
 	bool "64KB"
 
